### ISSUE TITLE
TypeError: Cannot read property 'sync' of undefined

### ISSUE THREAD

**Issue by brapp0111** [Ylianst/MeshCentral#4523]
**Describe the bug**
After entering login credentials, using Google login, I receive this error:
```
TypeError: Cannot read property 'sync' of undefined
    at handleStrategyLogin (/home/user/node_modules/meshcentral/webserver.js:2677:64)
    at Layer.handle [as handle_request] (/home/user/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/user/node_modules/express/lib/router/route.js:137:13)
    at /home/user/node_modules/meshcentral/webserver.js:6468:169
    at complete (/home/user/node_modules/passport/lib/middleware/authenticate.js:271:13)
    at /home/user/node_modules/passport/lib/middleware/authenticate.js:278:15
    at pass (/home/user/node_modules/passport/lib/authenticator.js:428:14)
    at Authenticator.transformAuthInfo (/home/user/node_modules/passport/lib/authenticator.js:450:5)
    at /home/user/node_modules/passport/lib/middleware/authenticate.js:275:22
    at /home/user/node_modules/passport/lib/http/request.js:41:7
    at /home/user/node_modules/passport/lib/sessionmanager.js:26:5
    at pass (/home/user/node_modules/passport/lib/authenticator.js:274:43)
    at serialized (/home/user/node_modules/passport/lib/authenticator.js:283:7)
    at /home/user/node_modules/meshcentral/webserver.js:6916:56
    at pass (/home/user/node_modules/passport/lib/authenticator.js:291:9)
    at Authenticator.serializeUser (/home/user/node_modules/passport/lib/authenticator.js:296:5)
```

**To Reproduce**
1. Go to Login Page
2. Click on Google Icon
3. See error

**Expected behavior**
Usually I would be logged in and see list of my machines.

**Screenshots**
 [New Recording - 9/7/2022, 11:21:18 AM](https://vimeo.com/747322845/4f1e285d44)

**Server Software (please complete the following information):**
 - OS: Ubuntu 20.04.5 LTS
 - Virtualization: None
 - Network: WAN, no reverse proxy, no cloudflare, no ssl offload]
 - Version: MeshCentral v1.0.82, WAN mode.
 - Node: v12.18.2

**Client Device (please complete the following information):**
 - Device: Desktop
 - OS: Windows 10
 - Network: Remote over WAN
 - Browser:  Google Chrome
 - MeshCentralRouter Version: N/A

**Remote Device (please complete the following information):**
N/A

**Additional context**

**Your config.json file**
```
{
  "__comment__" : "This is a sample configuration file, edit a section and remove the _ in front of the name. Refer to the user's guide for details.",
  "settings": {
    "Cert": "XXXXXXXXXXXXXXXXXXXXXXXX",
    "AgentCoreDump": true,
    "MongoDb": "mongodb://127.0.0.1:27017/meshcentral",
    "WANonly": true,
    "_LANonly": true,
    "_Minify": 1,
    "_SessionTime": 30,
    "_SessionKey": "MyReallySecretPassword1",
    "_DbEncryptKey": "MyReallySecretPassword2",
    "_DbExpire": {
      "events": 1728000,
      "powerevents": 864000
    },
    "Port": 443,
    "MpsPort": 4433,
    "RedirPort": 80,
    "_AllowLoginToken": true,
    "_AllowFraming": true,
    "_WebRTC": false,
    "_ClickOnce": false,
    "SelfUpdate": true,
    "_UserAllowedIP": "127.0.0.1,192.168.1.0/24",
    "_UserBlockedIP": "127.0.0.1,::1,192.168.0.100",
    "_AgentAllowedIP": "192.168.0.100/24",
    "_AgentBlockedIP": "127.0.0.1,::1",
    "_LocalDiscovery": {
      "name": "Local server name",
      "info": "Information about this server"
    },
    "_TlsOffload": true,
    "_MpsTlsOffload": true,
    "_No2FactorAuth": true,
    "_WebRtConfig": {
      "iceServers": [
        { "urls": "stun:stun.services.mozilla.com" },
        { "urls": "stun:stun.l.google.com:19302" }
      ]
    }
  },
  "domains": {
    "": {
      "Title": "MyServer",
      "Title2": "Servername",
      "_TitlePicture": "title-sample.png",
      "_UserQuota": 1048576,
      "_MeshQuota": 248576,
      "NewAccounts": 1,
      "authStrategies": {
        "__comment__": "This section is used to allow users to login using other accounts. You will need to get an API key from the services and register callback URL's",
        "google": {
          "_callbackurl": "https://XXXXXXXXXXXXXX/auth-google-callback",
          "newAccounts": true,
          "_newAccountsUserGroups": [ "ugrp//xxxxxxxxxxxxxxxxx" ],
          "clientid": "XXXXXXXXXXXXXXXXXXXX.apps.googleusercontent.com",
          "clientsecret": "XXXXXXXXXXXXXXXXXXXXXX"
        }
      },
      "_NewAccountEmailDomains": [ "sample.com" ],
      "Footer": "<a href='https://twitter.com/mytwitter'>Twitter</a>",
      "_CertUrl": "https://192.168.2.106:443/",
      "_PasswordRequirements": { "min": 8, "max": 128, "upper": 1, "lower": 1, "numeric": 1, "nonalpha": 1, "reset": 90, "force2factor": true },
      "_AgentNoProxy": true,
      "_GeoLocation": true,
      "_UserAllowedIP": "127.0.0.1,192.168.1.0/24",
      "_UserBlockedIP": "127.0.0.1,::1,192.168.0.100",
      "_AgentAllowedIP": "192.168.0.100/24",
      "_AgentBlockedIP": "127.0.0.1,::1",
      "__UserConsentFlags__" : "Set to: 1 for desktop, 2 for terminal, 3 for files, 7 for all",
      "_UserConsentFlags" : 7,
      "_Limits": {
        "_MaxUserAccounts": 100,
        "_MaxUserSessions": 100,
        "_MaxAgentSessions": 100,
        "MaxSingleUserSessions": 10
      },
      "_yubikey": { "id": "0000", "secret": "xxxxxxxxxxxxxxxxxxxxx", "_proxy": "http://myproxy.domain.com:80" },
      "_httpheaders": { "Strict-Transport-Security": "max-age=360000" },
      "_agentConfig": [ "webSocketMaskOverride=1" ],
      "_SessionRecording": {
        "_onlySelectedDeviceGroups": false,
        "_index": true,
        "_maxRecordings": 30,
        "_maxRecordingSizeMegabyets": 30,
        "_protocols": [1,2,200]
      }
    }
  },
  "letsencrypt": {
    "__comment__": "Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "XXXXXXXXXXXXX",
    "names": "XXXXXXXXXXXXX",
    "rsaKeySize": 3072,
    "production": true
  },
  "_peers": {
    "serverId": "server1",
    "servers": {
      "server1": { "url": "wss://192.168.2.133:443/" },
      "server2": { "url": "wss://192.168.1.106:443/" }
    }
  },
  "smtp": {
    "host": "smtp-relay.gmail.com",
    "port": 587,
    "from": "XXXXXXXXXXX",
    "tls": false
  }
}     
 ```

**Comment by si458** [comment]
please update to 1.0.85 as this has just been fixed

**Comment by brapp0111** [comment]
It is fixed by updating to 1.0.85.  Sorry I missed the previous bug report.
